programs/marinade-finance/src/instructions/delayed_unstake/claim.rs

```diff
@@ -103,12 +104,8 @@ impl<'info> Claim<'info> {
         }
 
         // If circulating_ticket_balance = sum(ticket.balance) is violated we can have a problem
```

Comment:
Please let me fix this part. checked_sub + saturating_sub is correct to be used in this case